include/lsst/afw/image/Pixel.h

```diff
@@ -570,24 +532,15 @@ ExprT1 operator+=(ExprT1& e1, ExprT2 e2) {
 //
 // The choice is made on the basis of std::is_arithmetic
```

Comment:
I don't think this comment is true anymore... `valid_operands` does not use `is_arithmetic`.